### PR TITLE
Facets component

### PR DESCRIPTION
**Pull request by xwatkins**
Displays a list of facets, and allows selection and highlight of selected. Styling will be reviewed later based on UX findings and UI improvements TBD.

### CODE REVIEW

File: src/components/facets.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+import '../styles/components/facets.scss';
+
+class Facets extends Component {
+  constructor(props) {
+    super(props);
+    this.isActive = this.isActive.bind(this);
+  }
+
+  isActive(facetName, facetValue) {
```
Comment (dlrice) [inline src/components/facets.jsx]: could be slightly more succinct with the equivalent
```
isActive(facetName, facetValue) {
     const { selectedFacets } = this.props;
     return facetName in selectedFacets && selectedFacets[facetName].includes(facetValue);
}
```

File: src/app/components.jsx [hunk 2]
```diff
@@ -127,6 +129,19 @@ const components = [
...
+    name: 'Facets',
+    component: Facets,
+    function: '',
+    purpose: '',
+    props: {
+      data: facetData,
+      selectedFacets: {
+        facet_2: ['value_2'],
+      },
+      toggleFacet: () => console.log('facet toggled'),
```
Comment (dlrice) [inline src/app/components.jsx]: Could be useful to console the facet name, value returned by the callback.

File: src/components/facets.jsx (hunk 1 above)
Comment (dlrice) [inline src/components/facets.jsx]: could be slightly more succinct with the equivalent
```
isActive(facetName, facetValue) {
     const { selectedFacets } = this.props;
     return facetName in selectedFacets && selectedFacets[facetName].includes(facetValue);
}
```

Comment (xwatkins) [reply src/components/facets.jsx]: duh! yeah 🥇

**Comment by xwatkins** [comment]
Ready for review

**Pull request merged**